app/v2/retrieveAccountingType/model/response/RetrieveAccountingTypeResponse.scala

```diff
@@ -16,13 +16,13 @@
 
 package v2.retrieveAccountingType.model.response
 
-import play.api.libs.json.{Json, OFormat}
+import play.api.libs.json.Reads
 import v2.common.models.AccountingType
 
 case class RetrieveAccountingTypeResponse(accountingType: AccountingType)
 
 object RetrieveAccountingTypeResponse {
 
-  implicit val format: OFormat[RetrieveAccountingTypeResponse] = Json.format[RetrieveAccountingTypeResponse]
-
+  implicit val reads: Reads[RetrieveAccountingTypeResponse] =
```

Comment:
This can be reverted back to OFormat like the below or split out into reads and Owrites if you prefer. An example of OFormat is the below: We don't care which type of business is used so we can just use `\\ ` as it is only these three that have `accountingType`. 

```
  implicit val format: OFormat[RetrieveAccountingTypeResponse] = OFormat(
    (JsPath \\ "accountingType").read[AccountingType].map(RetrieveAccountingTypeResponse.apply),
    (JsPath \ "accountingType").write[AccountingType].contramap[RetrieveAccountingTypeResponse](_.accountingType)
  )
```